README.md

```diff
@@ -198,6 +199,15 @@ A `TraceApi` object is guaranteed to be returned by both of these calls, even if
 
 A fully detailed overview of the `TraceApi` object is available [here](doc/trace-api.md).
 
+## How does automatic tracing work?
+
```

Comment:
I see both 'execution context' and 'request context for execution of a request' in consecutive sentences. I think, trying to read this from the perspective of someone who is completely unfamiliar with 'context', that this might be confusing enough to warrant either a rewording or a separate explanation on the difference.

Given that the title of this section is "How does automatic tracing work?" I'd suggest possibly reversing the order of points in this paragraph. For example, it could be written to first talk about what automatic tracing entails (in well-known framework and RPC modules, we automatically inject code to start, annotate, and end trace spans, grouped by request), then the challenge that this poses (that at any given execution context, esp. after crossing asynchronous boundaries such as callbacks and Promise.thens, it is not always possible to determine which request is being served), and then how we approach this problem now (use continuation-local-storage on top of async-listener to propagate information about the current request across asynchronous boundaries).